Gruntfile.js

```diff
@@ -32,6 +32,39 @@ module.exports = function(grunt) {
 			'wp-content/plugins/akismet/**',
 			'!wp-content/themes/twenty*/node_modules/**',
 		],
+
+		// All built CSS files, in /src or /build.
+		cssFiles = [
+			'wp-admin/css/*.min.css',
+			'wp-admin/css/*-rtl*.css',
+			'wp-includes/css/*.min.css',
+			'wp-includes/css/*-rtl*.css',
+			'wp-admin/css/colors/**/*.css',
+			'wp-includes/blocks/**/*.css',
+		],
+
+		// All built js files, in /src or /build.
+		jsFiles = [
+			'wp-admin/js/',
+			'wp-includes/js/',
+			'wp-includes/blocks/**/*.js',
```

Comment:
There are a few `*.js.map` files that could also be cleaned up when running `npm run build:dev`:
```suggestion
			'wp-includes/blocks/**/*.js',
			'wp-includes/blocks/**/*.js.map',
```
What do you think?